### PR TITLE
proposition of a call for contrib

### PR DESCRIPTION
**Pull request by Villetaneuse**
@Zimmi48 @thomas-lamiaux This is a first proposition for a call for contributions.
@Zimmi48 The images in the index are broken, I don't really know much about github hosted we pages, so I'll let you fix them. For the moment, they are in `src/images`.

### CODE REVIEW

File: ressources/call_for_contribution.txt [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Dear Coq users and developpers,
+
+We would like to shed some light on our new project "Coq Platform Doc".
+Our goal is to provide an online compilation of short, practical and interactive
+tutorials and how-to guides about Coq and the Coq Platform. We wish this content
+to be part of the next Rocq website. A detailed description of the project can
```
Comment (Zimmi48) [inline ressources/call_for_contribution.txt:6]:
```suggestion
to be part of the upcoming Rocq website. A detailed description of the project can
```

File: ressources/call_for_contribution.txt [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+be found in our Coq Enhancement Proposal (CEP) here:
+https://github.com/coq/ceps/pull/91
+
+The first few tutorials are available as well as  a (very preliminary) online
```
Comment (Zimmi48) [inline ressources/call_for_contribution.txt:10]:
```suggestion
The first few tutorials are available as well as a (very preliminary) online
```

File: ressources/call_for_contribution.txt [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+interface here:
+https://www.theozimmermann.net/platform-docs/
+At this moment, it contains:
+- a Search tutorial
+- a Basic library file and module management tutorial
+- 3 tutorials about the Equation plugin
+More will follow, especially with your help!
+
+There is a dedicated zulip channel to discuss and participate:
```
Comment (Zimmi48) [inline ressources/call_for_contribution.txt:19]:
```suggestion
There is a dedicated Zulip channel to discuss and participate:
```

File: ressources/call_for_contribution.txt [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+We would like to shed some light on our new project "Coq Platform Doc".
+Our goal is to provide an online compilation of short, practical and interactive
+tutorials and how-to guides about Coq and the Coq Platform. We wish this content
+to be part of the next Rocq website. A detailed description of the project can
+be found in our Coq Enhancement Proposal (CEP) here:
+https://github.com/coq/ceps/pull/91
+
+The first few tutorials are available as well as  a (very preliminary) online
+interface here:
+https://www.theozimmermann.net/platform-docs/
```
Comment (Zimmi48) [inline ressources/call_for_contribution.txt:12]:
```suggestion
https://www.theozimmermann.net/platform-docs/
(note that this is a temporary URL that will cease to be available
once the tutorials are migrated to the official Rocq website)
```

**Review by Zimmi48 — APPROVED** [review]

**Comment by Zimmi48** [comment]
> The images in the index are broken, I don't really know much about github hosted we pages, so I'll let you fix them. For the moment, they are in `src/images`.

Should be fixed with c610ecd43c5250680870bd37fb670e06ecdee7d7.

File: ressources/call_for_contribution.txt (hunk 4 above)
Comment (thomas-lamiaux) [reply ressources/call_for_contribution.txt:12]: It is already going to change when moved to a more official repo, right ?

Comment (Zimmi48) [reply ressources/call_for_contribution.txt:12]: Yes, but I was expecting the two events to be coordinated / happen at the same time.

Comment (Zimmi48) [reply ressources/call_for_contribution.txt:12]: Yes, @tabareau said we could expect to have the new website before the end of the year. But indeed, if we want to move the repo before that, we should be able to.

So my suggestion could become:

```suggestion
https://www.theozimmermann.net/platform-docs/
(note that this is a temporary URL that will cease to be available
once the tutorials are migrated to the official Coq organization)
```

**Comment by thomas-lamiaux** [comment]
I'am merging and posting it so that it is done

**Pull request merged**